src/views/components/create-model-form.jsx

```diff
@@ -96,8 +104,9 @@ export default class CreateModelForm extends React.Component {
                     <input type='text' value={this.state.location} onChange={this.changeLocation.bind(this)} placeholder='Location' /><br/>
                     Price per unit:<br/>
                     <input type='number' value={this.state.price} onChange={this.changePrice.bind(this)} placeholder='Price' /><br/>
-                    Amount in stock:<br/>
```

Comment:
We can't do this on the form right now, though, because a serialized model needs a number of items created with it.  If they create a new serialized model and default its count to 50, then we need to provide them with 50 item addresses so they can make barcodes and individually serialize each one.

I agree that the attributes would be nice for serialized models, but it is both out of scope of this story, and irrelevant to this form.